src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/WcfRestartService.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
This test has been changed to kill a dedicate servicehost instance instead of the whole process some time ago:-)